### DISCUSSION TITLE
World Size 1gb limit distributed between scenes?

### DISCUSSION THREAD

**Discussion by ckdlskgkdrhd** [naverz/zepeto-studio-global#55] [Q&A]
so I read how the limit of the world is 1gb
but can I use all full 1gb on 1 scene, and 1gb on 2nd scene, so on and forth.
or does it have to be the entire game(build game) into 1GB (yikes)

**Comment by naverz-mkao** [comment]
Hi,

Yes, unfortunately that 1GB limit applies to the entire build.

**Reply by ckdlskgkdrhd** [reply]
yikes..
